components/track/TrackItem.vue

```diff
@@ -48,6 +48,33 @@ function adjustHighlightText(highlight: Highlighting) {
     .replaceAll("**/", "</span>");
 }
 
+const fields = computed(() => {
+  const parts = [
+    props.track.title,
+    trackSongNumber(props.track),
+    props.track.meta.artist,
+    props.isTrackTypeKnown ? null : props.track.meta.album,
+  ];
+  const filtered = parts.filter((part) => part);
```

Comment:
We could use `parts.filter(Boolean)` here as well, but it is basically the same

Reply:
I don't understand your suggestion. Let me google that.

Reply:
got it. I didn't realize that your suggestion actually was valid code. Changed it